PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,34 @@
-A brief summary of your PR...
+Provide a brief summary of your PR...
 
-**Setup**
+## Tasks
 
-- [ ] A list of steps to get started with testing.
+- Link to the task in notion.
 
-**Related PRs**
+## Related PRs
 
 - Link to the related PR.
 
-**Tasks**
+## Notes
 
-- Link to the task in notion.
+- Any additional notes you think will be useful.
 
-**Testing**
+## Testing
```

Comment:
I don't really think we need the descriptions below, they would just be deleted with every PR? I'd prefer to keep this section strictly for steps we need to run when reviewing the PR.

Do you think it would be better to just add additional steps in the template and move those descriptions into a notion/github developer guide?

Also, I like the words `Verify` and `Testing` rather than `Recreate` and `Solution`, but that is just a personal preference 🤷‍♂ 

## Verify

- [ ] Pull down the latest `master` branch.
```shell
$ git checkout master
$ git pull --rebase
```
- [ ] Rebuild all containers.
```
$ c kc build && c kc start
```
- [ ] A list of steps to demonstrate the problem.

## Testing
- [ ] Checkout this branch.
```shell
$ git fetch --all
$ g checkout <insert branch name>
$ git pull --rebase
```
- [ ] Rebuild all containers.
```
$ c kc build && c kc start
```
- [ ] A list of steps to verify the solution.

Reply:
@allanchau, I keep getting PRs that don't have enough information, aren't completed or aren't thought through. I agree, I don't like the text that will be deleted all the time, but for now, I think we need it to lift the quality of the PR.

Reply:
I like the words verify and testing better too, I'll update it.